pkg/controllers/dynakube/extension/consts/consts.go

```diff
@@ -1,33 +1,14 @@
 package consts
 
-import "github.com/Dynatrace/dynatrace-operator/pkg/api"
-
 const (
-	ExtensionsAnnotationSecretHash = api.InternalFlagPrefix + "secret-hash"
+	// TODO rename without extra prefix use import alias
 
 	// secret
-	EecTokenSecretKey         = "eec.token"
-	EecTokenSecretValuePrefix = "EEC dt0x01"
-
-	OtelcTokenSecretKey         = "otelc.token"
-	OtelcTokenSecretValuePrefix = "dt0x01"
-
-	// shared volume name between eec and OtelC
-	ExtensionsTokensVolumeName = "tokens"
-
-	ExtensionsSecretConditionType  = "ExtensionsSecret"
-	ExtensionsServiceConditionType = "ExtensionsService"
-
-	ExtensionsControllerSuffix        = "-extensions-controller"
-	ExtensionsCollectorComPort        = 14599
-	ExtensionsCollectorTargetPortName = "collector-com"
-
-	ExtensionsSelfSignedTLSSecretSuffix     = "-extensions-controller-tls"
-	ExtensionsSelfSignedTLSCommonNameSuffix = "-extensions-controller.dynatrace"
+	TokenSecretKey         = "eec.token"
+	TokenSecretValuePrefix = "EEC dt0x01"
 
-	// TLSKeyDataName is the key used to store a TLS private key in the secret's data field.
-	TLSKeyDataName = "tls.key"
+	SecretConditionType  = "ExtensionsSecret"
+	ServiceConditionType = "ExtensionsService"
```

Comment:
do these need to be public?
can't they just "closer" where they are used?

Reply:
fixed in 1e7389728